frontend/courses/sample/draw/text.evy

```diff
@@ -0,0 +1,15 @@
+move 10 80
+text "Hello"
+
+move 10 60
+textsize 10
+text "all 🖖"
+
+move 10 40
+fontfamily "Luminari, fantasy"
```

Comment:
I dont have this font. Is this a mac thing? Perhaps you should stick to common web fonts?

Reply:
fantasy is the fallback. I searched for "common web fantasy fonts" and this is what came up, e.g.: https://blog.hubspot.com/website/web-safe-html-css-fonts or https://www.w3.org/Style/Examples/007/fonts.en.html

What does it display for you?

Reply:
I dont see the font changing at all. Just the same as the previous line (although it's a bit hard to tell for sure because the previous line is just "all", which is not much to compare with).

Reply:
That's kinda sad.
This is what it looks like for me:
<img width="361" alt="Screenshot 2023-04-24 at 1 02 22 PM" src="https://user-images.githubusercontent.com/1596871/233894196-b6f14aea-a5ff-4c16-ac4a-58c6747e013a.png">